packages/kit/src/runtime/server/endpoint.js

```diff
@@ -10,6 +10,11 @@ function error(body) {
 	};
 }
 
+/** @param {string} s */
```

Comment:
Sorry, I messed up, `unknown` is a more appropriate type than `string` here.

Reply:
guess I should have ran it locally :smile: